docs/dev/manual_testing.rst

```diff
@@ -1,5 +1,5 @@
 
-Manual Testing & QA
+Manual testing & QA
 ===================
 
 Accessibility (A11y) Testing
```

Comment:
There is  more recent document for testing in VitualBox, specific for Kolibri, instead of the link in line 54 here:
https://docs.google.com/document/d/1CG4Z0hofN0ipsny9mDf1xr2C_eY5c-T2nUbhlz2eZjA/edit?usp=sharing